scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -537,21 +538,30 @@ run_fsck() {
 
     # Try to mount the metadata device on /var/motr on both nodes
     if [[ $LOCAL_NODE_RECOVERY_STATE == $RSTATE3 ]]; then
-        if ! run_cmd_on_local_node "mount $LOCAL_MOTR_DEVICE $MD_DIR"; then
+        if run_cmd_on_local_node "mount $LOCAL_MOTR_DEVICE $MD_DIR"; then
+            # create directory "datarecovery" in /var/motr if not exist to store traces
+            run_cmd_on_local_node "mkdir -p $MD_DIR/datarecovery"
+        else
             run_cmd_on_local_node "mkfs.ext4 $LOCAL_MOTR_DEVICE"
```

Comment:
If the $LOCAL_MOTR_DEVICE was formatted before, this mkfs.ext4 will fail?
mkfs.ext4 --force?

Reply:
The script will not run mkfs.ext4 in normal path. The mkfs.ext4 on line # 545 will be executed only if the mount fails.

Reply:
It seems mkfs.ext4 does not check if the device has existing file system on the device or not.
mkfs.ext4 will directly mkfs on the target file with the trust that the user knows what he/she is doing.